index.html

```diff
@@ -13,18 +13,17 @@
 <body>
     <header>
         <nav>
-            <div><a href="index.html">Leo</a>
-            </div>
-            <div>
-                <div>
-                    <ul class="nav-items">
-                        <li><a class="nav-link" href="#">Portfolio</a></li>
-                        <li><a class="nav-link" href="#about">About</a></li>
-                        <li><a class="nav-link" href="#contact">Contact</a></li>
-                        <li><a class="nav-link" href="mailto: contact.popey17@gmail.com"><i class="fa-regular fa-envelope"></i></a></li>
-                    </ul>
-                </div>
-                <div><a href="#"><i class="fa-solid fa-bars"></i></a></div>
+            <a href="index.html">Leo</a>
+            <i class="fa-solid fa-bars menu-btn"></i>
+            <div class="nav-content">
+                <ul class="nav-items">
+                    <i class="fa-solid fa-xmark close-btn"></i>
+                    <li><a class="nav-link" href="#">Portfolio</a></li>
+                    <li><a class="nav-link" href="#about">About</a></li>
+                    <li><a class="nav-link" href="#contact">Contact</a></li>
+                    <li><a class="nav-link" href="mailto: contact.popey17@gmail.com"><i
```

Comment:
- [ ] [OPTIONAL] Good job implementing the previously required changes :hugs: However, the requirement asked that each navigation link should link to the respective section on your page - which is not working now, and that clicking on each link should close the menu so that users can see enjoy their excursion on your page :wink: Even though it was overlooked by my colleague, these functionalities will really improve your user experience, and also show recruiters how concerned you are with User Interface and User Experience.
- Here's a [screencast](https://www.awesomescreenshot.com/video/15359310?key=5faf528052ea1106299d90ea149e210e) of what it currently looks like.
  - [ ] To fixed the hyperlink issue, kindly add the `id` values that you have in your `<a href="#id"></a>` to the sections you are referencing them too.
  - Only that of the `Contact section` is working now. The value in your `href` for `About Section` is not consistent with the `#about-me` that you have in the About me while no `id` is attributed to `Work section`.